test/memspace_host.cpp

```diff
@@ -0,0 +1,74 @@
+// Copyright (C) 2024 Intel Corporation
+// Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "base.hpp"
+
+#include <umf/memspace.h>
+
+using umf_test::test;
+
+#define SIZE_4K (4096)
+
+struct predefinedMemspaceTest : test {
+
+    void SetUp() override {
+        ::test::SetUp();
+
+        hMemspace = umfMemspaceHostGet();
+        ASSERT_NE(hMemspace, nullptr);
```

Comment:
perhaps include `test_helpers.h` and use our `UT_ASSERT*` macros

Reply:
Done.